### PR TITLE
Prepare 2.15.0-rc.2 changelog

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This prepares the changelog for RC2, based on

* https://github.com/kubermatic/kubermatic/commit/7963161a42932e7ff3da17fe5052f7c36a665eaa
* https://github.com/kubermatic/dashboard/commit/57246f4273be5d839b5c543f8a867384dd04b1a2

When 2.15 is final, the changelogs for all RCs will be combined into a single 2.15.0 entry. For now we list the changes in between releases individually.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by xrstf** [comment]
/hold

Needs approval from CL and UI teams.

**Review by xmudrii — APPROVED** [review]
/lgtm
/hold
for other folks to take a look

**Pull request merged**
